### DISCUSSION TITLE
Googlebot doesn't see <Link>, how to fix it?

### DISCUSSION THREAD

**Discussion by vadymchornyidev** [vercel/next.js#51247] [Help] [answered]
### Summary

Hello everyone!

The problem is that I don't know how to do the right thing.
**I disable JavaScript and see that the links on the page are not working**.
That is, Googlebot cannot go to other pages of my site.

**How did I fix it? (but this is a bad option)**

I tried to replace Link with <a> and it so happened that when I disabled js, the links started working.

But the option to use <a> everywhere is very bad, so I wrapped the blocks with links in <Link> for site users and in <a> for the Google bot.

Can you tell me how to get rid of <a> altogether and switch to <Link> only?

Here is my code, which works now, but not correctly

```
<Link href={`/${company.Symbol}`}> // for users
   <div
       className={styles['ticket']}
       style={{ backgroundColor: TableColors[index] }}
   >
      <a href={`/${company.Symbol}`}>{company.Symbol}</a> // for google bot
   </div>
</Link>
```

### Additional information

I understand that <Link> works on the server side and I receive data from ssr.

**Comment by vadymchornyidev** [comment]
@icyJoseph Hello! Can you help me?

How to get rid of <a> so that Google bot can see links to other pages of the site.

Here is an example

This example works!

```
<a href={`/${company.Symbol}`}>
   <div
       className={styles['ticket']}
       style={{ backgroundColor: TableColors[index] }}
   >
       {company.Symbol}
   </div>
</a>
```

When I change <a> to <Link>, this option no longer works, and the Google bot does not see this link

**Reply by icyJoseph** [reply]
So the issue is that you have a Link from next/link wrapping an anchor tag? The next/link doesn't need a nested anchor tag in next 13.

What next version have you got?

**Reply by icyJoseph** [reply]
Somehow there's missing text in the comment, so I am not sure which one works and which one doesn't.

**Reply by icyJoseph** [reply]
@VadimChorrny I can't reproduce this, it works, as expected, with JavaScript disabled. What next.js version do you have? Can you show me the repository? You can add me to it, I can review, and comment back here, then you can kick me from it.

**Reply by vadymchornyidev** [reply]
> Somehow there's missing text in the comment, so I am not sure which one works and which one doesn't.

if I don't make an attachment with <a> googlebot won't see the link

This case works even if javascript is disabled

```
<a href={`/${company.Symbol}`}>
   <div
       className={styles['ticket']}
       style={{ backgroundColor: TableColors[index] }}
   >
       {company.Symbol}
   </div>
</a>
```

But this case didn't work, it is more correct to say that it works, but when I disable js, this link disappears

```
<Link href={`/${company.Symbol}`}>
   <div
       className={styles['ticket']}
       style={{ backgroundColor: TableColors[index] }}
   >
       {company.Symbol}
   </div>
</Link>
```

I use nextjs 13

**Reply by icyJoseph** [reply]
Could you take a screenshot of the HTML with JavaScript disabled and the Next Link?

**Reply by icyJoseph** [reply]
The link disappearing is super weird. If it's pre-rendered or SSRd it should be there. 

I also tested with next 13. Is this app Directory?

**Reply by vadymchornyidev** [reply]
yes, my code and screenshots

```
<Link href={`/${company.Symbol}`}>
    <div
        className={styles['ticket']}
        style={{ backgroundColor: TableColors[index] }}
    >
        {company.Symbol}
    </div>
</Link> // this link didn't work without js
```

![image](https://github.com/vercel/next.js/assets/65212516/dfd0b1c6-bd95-4292-8c4e-d5057fdc3561)

```
<a href={`/${company.Symbol}`}>
   <div
       className={styles['ticket']}
       style={{ backgroundColor: TableColors[index] }}
   >
      {company.Symbol}
   </div>
</a> // this code works fine without js and with him, but that's not <Link>
```

![image](https://github.com/vercel/next.js/assets/65212516/b3625978-8b5e-486a-9419-ea46baaa9cd4)

**Reply by icyJoseph** [reply]
And in the HTML that uses `Link`, only the div is shown? That's really wrong.

- How do you import the next/link?
- Could you show us the next.config.js file?

**Reply by icyJoseph** [reply, marked as answer]
@VadimChorrny I've managed to reproduce it, you must have accidentally set:

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  experimental: {
    newNextLinkBehavior: false,
  },
};

module.exports = nextConfig;
```

That will break it! Remove that `newNextLinkBehavior: false`, the default is true.
